### PR TITLE
chore: add display message when encountering login cache error

### PR DESCRIPTION
**Pull request by xiaolang124**
Add display message when encountering Azure account cache problem.

Fix: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_dashboards/dashboard/b7e7b190-a7fb-4c70-89f1-e63796c991c5

### CODE REVIEW

File: packages/vscode-extension/package.nls.json [hunk 1]
```diff
@@ -242,6 +242,7 @@
     "teamstoolkit.handlers.loginFailed": "Login failed, the operation is terminated.",
     "teamstoolkit.handlers.m365SignIn": "Successfully signed in to Microsoft 365 account.",
     "teamstoolkit.handlers.m365SignOut": "Successfully signed out of Microsoft 365 account.",
+    "teamstoolkit.handlers.loginCacheFailed": "Failed to get login account token from cache. You could try signing out and signing back in to your Azure Account using Teamsfx tree view or command palette",
```
Comment (swatDong) [inline packages/vscode-extension/package.nls.json:245]: Does this issue occur frequently? If true, consider adding help link/doc.

**Review by swatDong — APPROVED** [review]

File: packages/vscode-extension/package.nls.json (hunk 1 above)
Comment (xiaolang124) [reply packages/vscode-extension/package.nls.json:245]: In my experience, this only happens in rare cases. If I can find a better way to reproduce this, help link/doc may be added in later time.

Comment (HuihuiWu-Microsoft) [reply packages/vscode-extension/package.nls.json:245]: Is the wording 'Teamsfx' correct one or should we use 'Teams Toolkit'?

File: packages/vscode-extension/src/commonlib/azureLogin.ts [hunk 2]
```diff
@@ -65,23 +66,34 @@ class TeamsFxTokenCredential implements TokenCredential {
...
-        return {
-          token: token.accessToken,
-          expiresOnTimestamp: (tokenJson as any).exp * 1000,
-        };
+        return null;
       }
-    } else {
-      return null;
+    } catch (error) {
+      if ((error as any).message === "Entry not found in cache.") {
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/commonlib/azureLogin.ts:89]: Is this error message stable? For example, does this change in other languages?

File: packages/vscode-extension/package.nls.json (hunk 1 above)
Comment (xiaolang124) [reply packages/vscode-extension/package.nls.json]: Updated to 'Teams Toolkit'.

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by swatDong — APPROVED** [review]

**Pull request merged**